lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -249,14 +255,18 @@ def make_request(method)
         )
         nil
       rescue Faraday::SSLError
+        # if we've ended up here again, just abort
+        return nil unless check_ssl
```

Comment:
This will only be reachable if we have a bug, so I'm open to a less-silent way of handling this issue.